### DISCUSSION TITLE
Does UC mode only work on google chrome?

### DISCUSSION THREAD

**Discussion by Dylgod** [seleniumbase/SeleniumBase#2424] [Q&A] [answered]
I was wondering if UC Mode works on Edge since it is also a Chromium based browser.

**Comment by mdmintz** [comment, marked as answer]
**UC Mode** is only for **Chrome**, **Brave**, and **Opera**.

If using Brave or Opera, set the `binary_location` in `Driver()` or `SB()`.

On a Mac, those locations are:

Brave: `binary_location="/Applications/Brave Browser.app/Contents/MacOS/Brave Browser"`
Opera: `binary_location="/Applications/Opera.app/Contents/MacOS/Opera"`

On Windows and Linux, the locations will be different. You'll need to find the location and use it for `binary_location`.
